code/datums/martial/brassknuckles.dm

```diff
@@ -0,0 +1,31 @@
+/datum/martial_art/brassknuckles
+	name = "Krav Maga"
```

Comment:
i dont think we're in krav maga anymore

Reply:
oh god oh fuck he's reviewing a closed pr

Reply:
> 
> 
> oh god oh fuck he's reviewing a closed pr

I didnt notice it was closed until after I sent my review because, when have I ever checked to see if a PR was closed before reviewing it?